Assets/_Scripts/Systems/LevelSystem.cs

```diff
@@ -50,8 +50,6 @@ public enum LevelState
         protected override void Awake()
         {
             base.Awake();
-
-            ActiveSceneIndex = SceneManager.GetActiveScene().buildIndex;
```

Comment:
Pourquoi tu as enlevé ça (+ les autres en bas)? Ça risque de créer des problèmes si on sait pas quelle scène est active

(Par exemple, dans le PlayerReady.cs, je checkais si on était dans le lobby (pour ne pas que le Ready! show up dans la Game). Si on part le jeu du lobby et on ne sais pas que c'est la scène active, on ne pourra pas se mettre en ready)

Reply:
Le activesceneindex était uniquement a jour sur le host. C'est ce qui causais le problème de Ready!

Je l'ai enlevé parce qu'on a déjà une référence vers la scène actuel sans loadedscene. Le is lobby est encore là. Il se basse sur la scène qui load et pas un int qu'on garde cached localement.

Quand tu as testé que le ready marche, j'imagine que tu avais tester en commençant sur la scène lobby? Ce qui est explique le bon sceneindex même sur le client.

Reply:
Aaah makes sense

Reply:
les changements que tu as fais dans le levelSystem régler aussi le problème. Je ne vais pas changer inutilement la classe.